server/core/src/main/java/org/infinispan/server/core/admin/embeddedserver/CacheRemoveTask.java

```diff
@@ -0,0 +1,46 @@
+package org.infinispan.server.core.admin.embeddedserver;
+
+import org.infinispan.server.core.admin.AdminServerTask;
+
+/**
+ * Admin operation to remove a cache
+ * Parameters:
+ * <ul>
+ *    <li><strong>name</strong> the name of the cache to remove</li>
+ *    <li><strong>flags</strong> </li>
+ * </ul>
+ *
+ * @author Tristan Tarrant
+ * @since 9.1
+ */
+public class CacheRemoveTask extends AdminServerTask<Void> {
+
+   @Override
+   public String getTaskContextName() {
+      return "cache";
+   }
+
+   @Override
+   public String getTaskOperationName() {
+      return "remove";
+   }
+
+   @Override
+   public Void call() throws Exception {
+      if (isPersistent())
+         throw new UnsupportedOperationException();
+
+      String name = requireParameter("name");
+      cacheManager.executor().submitConsumer(localManager -> {
```

Comment:
this is the first simplistic implementation. Unfortunately there is no built-in way for a server node (which is handling the request) to talk to its domain controller. My plan is to use the wildfly remoting client from the server to the DC